modules/core/src/lib/widget-manager.ts

```diff
@@ -5,11 +5,16 @@ import type {MjolnirPointerEvent, MjolnirGestureEvent} from 'mjolnir.js';
 import type Layer from './layer';
 
 import {EVENTS} from './constants';
+import {deepEqual} from '../utils/deep-equal';
 
 export interface Widget<PropsT = any> {
+  id: string;
```

Comment:
Nit: Add JSDoc `/** */` comments to types and fields?

Reply:
Good catch. I've updated the docs as well.